### ISSUE TITLE
Some mob kills still don't count towards Deep Mob Learning

### ISSUE THREAD

**Issue by SonicSpeedster16** [47MarkIV/Absolute-Mek#59]
I noticed while playing that there are still a few mob kills that don't count towards Deep Mob Learning. I took the liberty to do a entity dump and go through and add them all to the Deep Mob Learning config and am posting the relevant section of my updated config (which I can verify works correctly) so you can use it in a future update.

```
##########################################################################################################
# data model mob names
#--------------------------------------------------------------------------------------------------------#
# Register entities that count towards leveling up the model
# Format is modname:entity_name
##########################################################################################################

"data model mob names" {
    # Blaze
    S:blaze <
        minecraft:blaze
        specialmobs:specialblaze
        specialmobs:cinderblaze
        specialmobs:conflagrationblaze
        specialmobs:emberblaze
        specialmobs:hellfireblaze
        specialmobs:infernoblaze
        specialmobs:joltblaze
        specialmobs:wildfireblaze
     >

    # Creeper
    S:creeper <
        minecraft:creeper
        netherex:spore_creeper
        specialmobs:darkcreeper
        specialmobs:deathcreeper
        specialmobs:dirtcreeper
        specialmobs:doomcreeper
        specialmobs:drowningcreeper
        specialmobs:endercreeper
        specialmobs:firecreeper
        specialmobs:jumpingcreeper
        specialmobs:lightningcreeper
        specialmobs:minicreeper
        specialmobs:splittingcreeper
     >

    # Dragon
    S:dragon <
        minecraft:ender_dragon
     >

    # Enderman
    S:enderman <
        minecraft:enderman
        deepmoblearning:trial_enderman
        farlanders:classic_enderman
        farlanders:mystic_enderman
        farlanders:mystic_enderminion
        farlanders:enderminion
        farlanders:rebel
        farlanders:farlander
        farlanders:elder_farlander
        farlanders:wanderer
        farlanders:ender_guardian
        farlanders:looter
        farlanders:titan
        farlanders:ender_golem
        farlanders:fanmade_enderman
        specialmobs:specialenderman
        specialmobs:blindingenderman
        specialmobs:icyenderman
        specialmobs:lightningenderman
        specialmobs:minienderman
        specialmobs:mirageenderman
        specialmobs:mirageenderman
        specialmobs:thiefenderman
        
        
     >

    # Ghast
    S:ghast <
        minecraft:ghast
        netherex:ghast_queen
        netherex:ghastling
        specialmobs:specialghast
        specialmobs:babyghast
        specialmobs:fighterghast
        specialmobs:kingghast
        specialmobs:queenghast
        specialmobs:unholyghast
     >

    # Guardian
    S:guardian <
        minecraft:elder_guardian
        minecraft:guardian
     >

    # Matter Overdrive Androids
    S:mo_android <
        matteroverdrive:ranged_rogue_android
        matteroverdrive:rogue_android
     >

    # Shulker
    S:shulker <
        minecraft:shulker
     >

    # Skeleton
    S:skeleton <
        minecraft:stray
        minecraft:skeleton
        mekanism:babyskeleton
        rewired:cyberskeleton
        specialmobs:specialskeleton
        specialmobs:bruteskeleton
        specialmobs:fireskeleton
        specialmobs:gatlingskeleton
        specialmobs:giantskeleton
        specialmobs:knightskeleton
        specialmobs:ninjaskeleton
        specialmobs:poisonskeleton
        specialmobs:sniperskeleton
        specialmobs:spitfireskeleton
        specialmobs:strayskeleton
     >

    # Slime
    S:slime <
        minecraft:slime
        minecraft:magma_cube
        deepmoblearning:trial_slime
        specialmobs:blackberryslime
        specialmobs:blueberryslime
        specialmobs:caramelslime
        specialmobs:flyinglavaslime
        specialmobs:grapeslime
        specialmobs:hardenedlavaslime
        specialmobs:lemonslime
        specialmobs:speciallavaslime
        specialmobs:specialslime
        specialmobs:stickylavaslime
        specialmobs:strawberryslime
        specialmobs:volatilelavaslime
        specialmobs:watermelonslime
     >

    # Spider
    S:spider <
        minecraft:spider
        minecraft:cave_spider
        deepmoblearning:trial_spider
        deepmoblearning:trial_cave_spider
        netherex:coolmar_spider
        specialmobs:specialspider
        specialmobs:specialcavespider
        specialmobs:babyspider
        specialmobs:desertspider
        specialmobs:flyingspider
        specialmobs:giantspider
        specialmobs:hungryspider
        specialmobs:motherspider
        specialmobs:palespider
        specialmobs:poisonspider
        specialmobs:webspider
        specialmobs:witchcavespider
        specialmobs:witchspider
        specialmobs:flyingcavespider
        specialmobs:mothercavespider
     >

    # Tinker construct slime
    S:tinkerslime <
        tconstruct:blueslime
     >

    # Witch
    S:witch <
        minecraft:witch
        specialmobs:specialwitch
        specialmobs:dominationwitch
        specialmobs:shadowswitch
        specialmobs:undeadwitch
        specialmobs:wildswitch
        specialmobs:windwitch
        specialmobs:undeadwitch
     >

    # Wither
    S:wither <
        minecraft:wither
     >

    # Wither Skeleton
    S:witherskeleton <
        minecraft:wither_skeleton
        specialmobs:specialwitherskeleton
        specialmobs:brutewitherskeleton
        specialmobs:gatlingwitherskeleton
        specialmobs:giantwitherskeleton
        specialmobs:knightwitherskeleton
        specialmobs:ninjawitherskeleton
        specialmobs:sniperwitherskeleton
        specialmobs:spitfirewitherskeleton
     >

    # Zombie
    S:zombie <
        minecraft:husk
        minecraft:zombie
        minecraft:zombie_villager
        minecraft:zombie_pigman
        cyberware:cyberzombie
        specialmobs:specialzombie
        specialmobs:specialpigzombie
        specialmobs:huskzombie
        specialmobs:brutezombie
        specialmobs:brutepigzombie
        specialmobs:firezombie
        specialmobs:fishingzombie
        specialmobs:fishingpigzombie
        specialmobs:giantzombie
        specialmobs:giantpigzombie
        specialmobs:hungryzombie
        specialmobs:hungrypigzombie
        specialmobs:huskzombie
        specialmobs:plaguezombie
        specialmobs:plaguepigzombie
        specialmobs:knightpigzombie
        specialmobs:vampirepigzombie
     >
}
```

**Comment by 47MarkIV** [comment]
Sorry, I cannot tell what changed without a line by line compare. Declined

**Comment by SonicSpeedster16** [comment]
@47MarkIV sorry I should have been more specific in what I changed. Here's the details in case you'd still be interested in using this for a possible future release:

1. Removed Baby Mobs entities (since the mod was removed in a previous update).
2. Re-added the Baby Skeleton (which is added by Mekanism).
3. Fixed the entity name for the Fan Made Enderman (currently they don't count towards the Enderman data model).
4. Added the Cyber Skeletons from ReWIRED.
5. Added mobs from NetherEX (Spore Creeper, Coolmar Spider & 2 Ghast variants).
6. Added some missing Pigmen, Witch, and Slime variants from Special Mobs.
